src/plugins/theme/spacing.js

```diff
@@ -47,43 +50,91 @@ module.exports = function () {
     28: '2.8rem',
     29: '2.9rem',
     30: '3rem',
+    31: '3.1em',
     32: '3.2rem',
+    33: '3.3rem',
     34: '3.4rem',
+    35: '3.5rem',
     36: '3.6rem',
+    37: '3.7rem',
     38: '3.8rem',
+    39: '3.9rem',
     40: '4rem',
+    41: '4.1rem',
+    42: '4.2rem',
+    43: '4.3rem',
+    44: '4.4rem',
     45: '4.5rem',
+    46: '4.6rem',
+    47: '4.7rem',
     48: '4.8rem',
+    49: '4.9rem',
```

Comment:
As much as we'd like to stick to the modular spacers, we very frequently need these extra values (esp below 50px). Sometimes to compensate for optical effects, or for finetuning positions of elements.